.evergreen/run-ocsp-cache-test.sh

```diff
@@ -30,40 +30,53 @@
 #
 # Environment variables:
 #
-# CDRIVER_ROOT
-#   Optional. The path to mongo-c-driver source (may be same as CDRIVER_BUILD).
-#   Defaults to $(pwd)
-# CDRIVER_BUILD
-#   Optional. The path to the build of mongo-c-driver (e.g. mongo-c-driver/cmake-build).
-#   Defaults to $(pwd)
 # CERT_TYPE
 #   Required. Set to either RSA or ECDSA.
 
-set -o errexit  # Exit the script with error if any of the commands fail
+set -o errexit
+set -o pipefail
 
-CDRIVER_ROOT=${CDRIVER_ROOT:-$(pwd)}
-CDRIVER_BUILD=${CDRIVER_BUILD:-$(pwd)}
+to_absolute() (
+  cd "${1:?}" && pwd
+)
 
-if [ -z "$CERT_TYPE" ]; then
-    echo "Required environment variable 'CERT_TYPE' unset. See file comments for help."
-    exit 1;
-fi
+to_windows_path() {
+  cygpath -aw "${1:?}"
+}
+
+print_var() {
+  printf "%s: %s\n" "${1:?}" "${!1:-}"
+}
+
+check_var_req() {
+  : "${1:?}"
+  : "${!1:?"required variable ${1} is unset or null!"}"
+  print_var "${1}"
+}
```

Comment:
This set of incantations, and slight variations thereof, appear frequently. Could it be refactored into a shared file that is dot-sourced by the others?